bin/ansible-connection

```diff
@@ -33,6 +35,21 @@ from ansible.utils.display import Display
 from ansible.utils.jsonrpc import JsonRpcServer
 
 
+@contextmanager
+def file_lock(lock_path):
+    """
+    Uses contextmanager to create and release a file lock based on the
```

Comment:
I think it's obvious from the decorator that it's a CM.
I'd be nicer to read smth like:
```
Acquire a lock at a given path and clean it up upon release.
```